python/dgl/transform.py

```diff
@@ -383,6 +384,107 @@ def reverse(g, share_ndata=False, share_edata=False):
         g_reversed._edge_frame = g._edge_frame
     return g_reversed
 
+def reverse_heterograph(g, share_ndata=True, share_edata=False):
```

Comment:
Why don't we directly change `dgl.reverse`?

Reply:
For compatablity.

Reply:
Then, why don't we create a transform branch and make the PR into dmlc:transform?